lte/gateway/c/oai/tasks/sgw/spgw_state.cpp

```diff
@@ -82,7 +82,8 @@ void sgw_free_s11_bearer_context_information(
   if (*context_p) {
     sgw_free_pdn_connection(
         &(*context_p)->sgw_eps_bearer_context_information.pdn_connection);
-
+    clear_protocol_configuration_options(
+        &(*context_p)->sgw_eps_bearer_context_information.saved_message.pco);
```

Comment:
how come we have not seen this memory leak before?

Reply:
What we had till now in PCO options were all container IDs. For container ID, the request does not include any data. Data is included only in the response. But ipcp is a protocol id and in this case request also contains data and needs to be freed